src/status_im/ui/screens/main_tabs/views.cljs

```diff
@@ -1,58 +1,65 @@
 (ns status-im.ui.screens.main-tabs.views
-  (:require [status-im.ui.components.drawer.view :refer [drawer-view]]
-            [status-im.ui.components.icons.vector-icons :as vector-icons]
+  (:require-macros [status-im.utils.views :as views])
+  (:require [status-im.ui.components.icons.vector-icons :as vector-icons]
             [status-im.ui.components.react :as react]
-            [status-im.ui.components.status-bar.view :refer [status-bar]]
-            [status-im.ui.components.styles :as styles]
-            [status-im.ui.components.tabs.styles :as tabs.styles]
-            [status-im.ui.components.tabs.views :as tabs]
+            [status-im.ui.components.status-bar.view :as status-bar.view]
+            [status-im.ui.components.styles :as common.styles]
             [status-im.i18n :as i18n]
-            [status-im.ui.screens.chats-list.views :refer [chats-list]]
-            [status-im.ui.screens.contacts.views :refer [contact-groups-list]]
-            [status-im.ui.screens.discover.views :refer [discover]]
-            [status-im.ui.screens.wallet.main.views :refer [wallet]])
-  (:require-macros [status-im.utils.views :as views]))
+            [status-im.ui.screens.chats-list.views :as chats-list.views]
+            [status-im.ui.screens.profile.views :as profile.views]
+            [status-im.ui.screens.wallet.main.views :as wallet.views]
+            [status-im.ui.screens.main-tabs.styles :as styles]
+            [re-frame.core :as re-frame]))
+
+(def tabs-list-data
+  {:chat-list
```

Comment:
Any particular reason for this change? Also `view-id` can probably be removed and replaced by the key value.

Reply:
what change do you mean? changed vector to map? yeah you're right